package.json

```diff
@@ -49,7 +50,14 @@
     "debug:build:prod": "node-nightly --inspect --debug-brk node_modules/webpack/bin/webpack.js  --config webpack.prod.config.ts",
     "docs": "typedoc --options typedoc.json ./src",
     "lint": "tslint \"src/**/*.ts\" || true",
-    "global": "npm install -g angular-cli nodemon npm-check-updates rimraf ts-node typedoc typescript webpack webpack-bundle-size-analyzer rollup marked node-gyp"
+    "global": "npm install -g angular-cli nodemon npm-check-updates rimraf ts-node typedoc typescript webpack webpack-bundle-size-analyzer rollup marked node-gyp protractor",
+    "postglobal": "npm link protractor",
```

Comment:
Because as it is we have no information about the version of protractor that this project is dependent on. 

If later, for a different project you need to install a different version of protractor, globally, then things in this project can start breaking, and since you haven't changed anything to this project, that'll be hard to debug. 

If this project has a local dependency on protractor, we'll know that that version will work, and we always use its executable instead of the global one, by using its path in the `node_modules` folder.

To prevent those kinds of issues we should use as few as possible global dependencies. So if you want to only have a single install of protractor, I'd leave out the global dependency, add it as a local dependency and use `node_modules/protractor/bin/protractor` as the executable in the npm tasks instead